installation/troubleshoot_installer.md

```diff
@@ -293,3 +294,18 @@ This error will be logged, in `<GO_LOG_DIR>/go-server.out.log` on server and
 
 Resolution: Update the environment variable JAVA_HOME set in `/etc/default/go-server`, `/etc/default/go-agent`
 to Java 8 path, and restart the Go server/agent.
+
+<a name="ports-in-use"></a>
+### Port 8153 (HTTP) or 8154 (HTTPS) could not be opened
+
+This issue shows up an error when starting GoCD Server:
+
+    Port 8153 could not be opened. Please Check if it is available
+    Port 8154 could not be opened. Please Check if it is available
+
+In order to change default ports, edit `go-server.default` file, which is located in GoCD Server root directory:
```

Comment:
Thanks @arvindsv! I didn't installed GoCD Server, but I run it via zip. I've added another commit explaining it.